cmake/support_catalyst.cmake

```diff
@@ -22,7 +22,9 @@ macro(FindCatalyst target_name)
         message(INFO " Building against local Catalyst - path: ${LIGHTNING_CATALYST_SRC_PATH} - GIT TAG: ${CATALYST_GIT_TAG}")
 
         target_include_directories(${target_name} PUBLIC ${LIGHTNING_CATALYST_SRC_PATH}/runtime/lib/backend/common)
+        target_include_directories(${target_name} PUBLIC ${LIGHTNING_CATALYST_SRC_PATH}/runtime/lib/capi)
```

Comment:
Backend devices should only depends on `runtime/include`. 

```
        target_include_directories(${target_name} PUBLIC ${LIGHTNING_CATALYST_SRC_PATH}/runtime/lib/capi)
        target_include_directories(${target_name} PUBLIC ${LIGHTNING_CATALYST_SRC_PATH}/runtime/tests)
```

Could you remove `capi` and `tests` dirs from the list of targets?

Reply:
Done